_posts/2020-05-11-BART-Summary.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "BART: Are all pretraining techniques created equal?"
+author: AntonioLopardo
+modified:
+comments: true
+tags: ""
+image:
+  thumb: BART_Board_cropped_more.jpg
+---
+<script type="text/javascript" async
+  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML">
+</script>
+
+![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FAntonioLprd%2FMp23o_Xx8j.jpg?alt=media&token=72dbb3c1-93a5-4b9d-8cec-3e006952568e)
+
+> Paper summary: BART: Denoising Sequence-to-Sequence Pre-training for Natural Language Generation, Translation, and Comprehension , Oct. 2019. ([link](https://arxiv.org/abs/1910.13461))
+
+## **Why is this important?**
+Lewis and et al. in this paper present valuable comparative work on different pre-training techniques and show how this kind of work can be used to guide large pre-training experiments reaching SOTA results. 
```

Comment:
I think it's generally a good idea to introduce abbreviations before using them. In this case I suggest replacing _SOTA_ with _state-of-the-art (SOTA)_. Makes articles like this approachable to everyone! :D